tests/parametric/test_trace_sampling.py

```diff
@@ -0,0 +1,232 @@
+import pytest
+from utils.parametric.spec.trace import Span
+from utils.parametric.spec.trace import find_span_in_traces
+import json
+from utils import coverage, rfc, scenarios
+from utils.parametric.spec.trace import SAMPLING_PRIORITY_KEY, SAMPLING_RULE_PRIORITY_RATE, SAMPLING_LIMIT_PRIORITY_RATE
+
+
+@scenarios.parametric
+class Test_Trace_Sampling_Basic:
```

Comment:
Can you add the link to the RFC on each class ? 

```suggestion
@rfc("link-to-the-rfc")
class Test_Trace_Sampling_Basic:
```

Reply:
Thanks! I've added it. The basic scenarios where the part of another RFC and I didn't have it at hand at time of the PR creation.